From 1ee00df22fe164157bc5d27dfcb6dc312cf94d3d Mon Sep 17 00:00:00 2001
From: aaronw <aaron.w2@gmail.com>
Date: Thu, 23 May 2019 17:11:53 -0700
Subject: [PATCH 0913/1239] mmc: octeontx2: CNF95XX Fix timing taps

This fixes the CMD_OUT_TAP and DATA_OUT_TAP for the CNF95XX
OcteonTX2 processor.

Change-Id: I257b6dc8d58f8c2e74c4ec172b5fe489a522c04c
Signed-off-by: aaronw <aaron.w2@gmail.com>
Reviewed-on: https://sj1git1.cavium.com/9840
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
---
 drivers/mmc/octeontx_hsmmc.c | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/drivers/mmc/octeontx_hsmmc.c b/drivers/mmc/octeontx_hsmmc.c
index 9a70b25ebe..d0f5519247 100644
--- a/drivers/mmc/octeontx_hsmmc.c
+++ b/drivers/mmc/octeontx_hsmmc.c
@@ -1775,12 +1775,13 @@ static int octeontx_mmc_configure_delay(struct mmc *mmc)
 			case MMC_HS_52:
 			case MMC_DDR_52:
 			case MMC_HS_200:
-				cout = 10;
-				dout = 5;
+				cout = is_board_model(CN95XX) ? 10 : 5;
+				/*cout = 10;*/
+				dout = is_board_model(CN95XX) ? 10 : 5;
 				break;
 			case MMC_HS_400:
-				cout = 5;
-				dout = 5;
+				cout = is_board_model(CN95XX) ? 10 : 5;
+				dout = is_board_model(CN95XX) ? 10 : 5;
 				break;
 			default:
 				pr_err("%s(%s): Invalid mode %d\n", __func__,
-- 
2.29.0

